### ISSUE TITLE
Node is executable frontend

### ISSUE THREAD

**Issue by lastmeta** [lastmeta/Satori#5]
Make a node js app and Run the light client in the node js app:
https://streamr.network/docs/streamr-network/using-a-light-node

Make the python app executable
https://cx-freeze.readthedocs.io/en/latest/setup_script.html#commands

Build an API between the data module and the node app or port the data module into the node app.

Make the node app executable
https://medium.com/jspoint/how-to-create-an-executable-exe-file-from-javascript-code-using-node-js-45154ba4de20

and make it run the model manager python executable.

The reason we have to do this is we can't run a broker node because we don't want to make the user download docker. So we need to use the light client which requires node.

**Comment by lastmeta** [comment]
Streamr is giving me trouble https://discord.com/channels/801574432350928907/879259412743716875/950414855683321896. Of course this exploratory jaunt into streamr was a pre mature optimization. The correct way to make this is as a series of microservices that talk to each other with an API for each. We shouldn't care, at this point what the front end is or how it's written, nor should we care about what network of streams we connect to. Instead we should design the api for those modular components to talk to, make an ear that we listen on. So we should return to the connection between the data manager and the model manager. and we can spoof our own network programmatically or make an actual centralized stream network using elixir eventually.

**Comment by lastmeta** [comment]
We'll probably put the front end UI on the flask app actually. that way we can see model stats more easily

**Comment by lastmeta** [comment]
created the reactive structure, and the UI, as well as a streamr spoof to test it all out. We should be able to come back to this soon.

**Comment by fonty1** [comment]
> 1. will my plan to use the Light client require any advanced setups from the users on whose computers it will run? (such as opening ports, running it in a docker etc). I assumed the entire reason there is a light client is to remove any advance setup requirements, but I want to verify that.

Yes, that’s right - the light node is a simple dependency that can be included into any JavaScript application. No advanced 

For small stream topologies behind difficult NATs there might be difficulties connecting in a peer-to-peer manner. Having a light or broker node on the open internet, subscribed to the stream can help.

> 2. I'm sure you can use the light client to subscribe to streams, but can you use it to publish data points to a new stream? furthermore, can you use it to publish a brand new stream?

Yes, publishing and subscribing can be done with the light node (streamr client). You can also create new streams and publish/subscribe to those as much as you want. Creating a stream costs some MATIC at the moment (writing to the stream registry on Polygon)
